docs/mkdocs.yml

```diff
@@ -21,13 +135,17 @@ nav:
       - UDCs: reference/udc.md
   - Appendix:
       - Earthly: appendix/earthly.md
-markdown_extensions:
-  - admonition
-  - pymdownx.details
-  - pymdownx.superfences:
-      custom_fences:
-        - name: mermaid
-          class: mermaid
-          format: !!python/name:pymdownx.superfences.fence_code_format
-  - pymdownx.tabbed:
-      alternate_style: true
+      - Documentation Examples:
+          - Examples: appendix/examples/index.md
+          - Abbreviations: appendix/examples/abbreviations.md
+          - Admonitions: appendix/examples/admonitions.md
+          - Code Formatting: appendix/examples/code-formatting.md
+          - Mermaid Diagrams: appendix/examples/mermaid-diagrams.md
+          - Kroki Diagrams: appendix/examples/kroki-diagrams.md
+      - Important Information:
```

Comment:
How did you get icons in this section? It looks kind of weird that the documentation examples and important information both have icons, but nothing else does.

Reply:
I think the icons really enhance the look of the docs.
It's not hard to add them.
I will fix the TOC issue you mentioned above first, and then look at adding icons to at least the top level pages.
The icon is added using a bit on yml frontmatter to the markdown, or inside a special file which sets yml frontmatter metadata for all files beside and below it.

Reply:
Yeah, I think adding icons to the others would be fine. It just looks funny when some have it and some don't.

Reply:
Done